app/services/stats/etl/closed_cases.rb

```diff
@@ -2,58 +2,14 @@
 
 module Stats
   module ETL
-    class ClosedCases
-      attr_reader :results_filepath, :retrieval_scope
-
-      ROWS_PER_FRAGMENT = 1000 # Arbitrary value, may require experimentation
+    class ClosedCases < BaseClosedCases
       RESULTS_NAME = 'closed-cases'.freeze
 
-      # +retrieval_scope+ : ActiveQuery relation which Query::CaseReport
-      # will use to ensure Warehouse retrieval is scoped for the current
-      # requesting user
-      def initialize(retrieval_scope:)
-        @retrieval_scope = retrieval_scope
-        @current_fragment_num = 0
-
-        self
-          .extract
-          .transform
-          .load
-      end
-
-      def extract
-        # We use `@_temp_files` to prevent the Temp files being cleared
-        # by ruby GC before they have been fully processed. In EC2 instances
-        # this happens rapidly. Manual clearup/purge required as a result.
-        #
-        # Ordering of generation matters, each function call creates a temp CSV
-        @_temp_files = generate_header_fragment + generate_data_fragments
-        self
-      end
 
-      def transform
-        self
+      def result_name
+        RESULTS_NAME
```

Comment:
Super tiny thing, but could you clarify what this method's for?

The fact that it has both plural and singlular forms of "result" here caught my eye

Reply:
the method is return the file name of csv zip file (used in func: transform()),  the original code uses the "result_name",  I changed the constant name to keep consistent